app/Http/Controllers/Admin/PayloadHistoryController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Enums\PayloadHistoryStatus;
+use App\Repositories\Interfaces\WebhookRepositoryInterface as WebhookRepository;
+use App\Repositories\Interfaces\MessageHistoryRepositoryInterface as MessageHistoryRepository;
+use App\Repositories\Interfaces\PayloadHistoryRepositoryInterface as PayloadHistoryRepository;
+
+class PayloadHistoryController extends Controller
+{
+    private $payloadHistoryRepository;
+    private $webhookRepository;
+    private $messageHistoryRepository;
+
+    public function __construct(
+        PayloadHistoryRepository $payloadHistoryRepository,
+        MessageHistoryRepository $messageHistoryRepository,
+        WebhookRepository $webhookRepository
+    ) {
+        $this->payloadHistoryRepository = $payloadHistoryRepository;
+        $this->webhookRepository = $webhookRepository;
+        $this->messageHistoryRepository = $messageHistoryRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $searchParams = $request->search;
+        $perPage = config('paginate.perPage');
+        $payloadHistories = $this->payloadHistoryRepository->getAllAndSearch($perPage, $searchParams);
+        $webhooks = $this->webhookRepository->getAll()->pluck('id', 'name');
+        $payloadHistoryStatuses = PayloadHistoryStatus::toArray();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/PayloadHistoryController.php</br>**Line**: 41</br>No space found after comma in function call